shopfloor/services/zone_picking.py

```diff
@@ -478,9 +488,33 @@ def _scan_source_package(self, barcode):
         move_lines = self._find_location_move_lines(package=package)
         if move_lines:
             response = self._response_for_set_line_destination(first(move_lines))
+            return response, message
+        if package.location_id:
+            # Check if the package selected can be a substitute on a move line
+            move_lines = self._find_location_move_lines(
+                locations=package.location_id,
+                product=package.product_packaging_id.product_id,
+            )
+        if move_lines:
```

Comment:
shouldn't this block be indented under `if package.location_id`?

Reply:
No, if does not have too, but the return case in it can be removed